docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2087,7 +2087,7 @@ The path to the file containing the hostnames, ports, and SHA256 certificate fin
     ```
 
 The URL on which the
-[Orion node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is
+[Tessera node](../../Tutorials/Privacy/Configuring-Privacy.md#4-create-orion-configuration-files) is
```

Comment:
this still has create-orion-configuration-files

Reply:
Fixed